test/integration/driver.go

```diff
@@ -215,18 +204,33 @@ func makeRequests(pods map[string]string) map[string][]string {
 		for dst := range pods {
 			for _, port := range []string{"", ":80", ":8080"} {
 				for _, domain := range []string{"", "." + namespace} {
-					url := fmt.Sprintf("http://%s%s%s/%s", dst, domain, port, src)
-					log.Printf("Making a request %s from %s...\n", url, src)
-					request := shell(fmt.Sprintf("kubectl exec %s -c app client %s", pods[src], url))
-					log.Println(request)
-					match := regexp.MustCompile("X-Request-Id=(.*)").FindStringSubmatch(request)
-					if len(match) > 1 {
-						id := match[1]
-						log.Printf("id=%s\n", id)
-						out[src] = append(out[src], id)
-						out[dst] = append(out[dst], id)
-					} else if src != "t" || dst != "t" {
-						log.Fatalf("Failed to inject proxy from %s to %s (url %s)", src, dst, url)
+					n := 1
+					for {
```

Comment:
The initialization and increment steps could be pulled into the for clause, i.e. "for (n := 1; ; n++)"